scripts/functions.sh

```diff
@@ -8,8 +8,11 @@ function datef() {
 
 function createConfig() {
     cd "$APP_PERSIST_DIR"
-    CLIENT_ID="$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)"
+    ARG1=$1
+    DEFAULT_ID="$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)"
+    CLIENT_ID="${ARG1:=$DEFAULT_ID}"
     CLIENT_PATH="$APP_PERSIST_DIR/clients/$CLIENT_ID"
+    [ -d $CLIENT_PATH ] && CLIENT_PATH=${CLIENT_PATH}_1
```

Comment:
I think if given path already exists it's better to return an error code. Otherwise, we will end up in a situation where we have two certificates with the same subject name (CLIENT_ID is present in generated certificate in subject field), which in its turn may lead to other problems. For instance, when we remove client with `rmclient.sh` we supply subject name as a parameter and certificate is revoked.